client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -43,14 +44,17 @@ export const QuickLinks = ( {
 	trackAddEmailAction,
 	trackAddDomainAction,
 	isExpanded,
-	expand,
-	collapse,
+	updateHomeQuickLinksToggleStatus,
 	isUnifiedNavEnabled,
 	siteAdminUrl,
 	editHomePageUrl,
 	siteSlug,
 } ) => {
 	const translate = useTranslate();
+	const debouncedUpdateHomeQuickLinksToggleStatus = debounce(
```

Comment:
Another option would be to define a `debouncedSavePreference` in the file scope. That would remove the debouncing logic from the component and mean we wouldn't be re-initialising the debounced function every time the component function is called.
Perhaps a micro optimisation 🤷

Reply:
```js
const debouncedSavePreference = debounce(savePreference, 1000);

const mapDispatchToProps = {
  ...
  expand: () => debouncedSavePreference('homeQuickLinksToggleStatus', 'expanded');
  collapse: () => debouncedSavePreference('homeQuickLinksToggleStatus', 'collapse');
}
```
Yeah, I tried to use `debouncedSavePreference` as above, but it would get the error for our `wpcom-api-middleware`
<img width="300" src="https://user-images.githubusercontent.com/13596067/128116820-c141847b-6cc4-4b01-a583-dcf5905ae96a.png" />

I don't dive into the problem, so I just keep this way

Reply:
Ah I see now, the functions in `mapDispatchToProps` must be "action creators", but `debouncedSavePreference()` isn't an action creator because it returns `void`. So the timing functions have to be done in the component.